websock/websock.nim

```diff
@@ -229,9 +229,15 @@ proc connect*(
   if uri.port.len <= 0:
     uri.port = if secure: "443" else: "80"
 
+  let pathQuery =
```

Comment:
would `$uri` work here? It should reconstruct the whole uri (including logins etc)